Sources/UILabel.swift

```diff
@@ -48,7 +48,8 @@ open class UILabel: UIView {
         layer.contentsGravity = textAlignment.contentsGravity()
     }
 
-    open var font: UIFont = .systemFont(ofSize: 16) {
+    static var defaultFont = UIFont.systemFont(ofSize: 16)
+    open var font: UIFont = UILabel.defaultFont {
```

Comment:
I noticed that the crash actually comes from the default here

```swift
let label = UILabel(frame: frame)
label.font = NoteName.font // already static
```

Reply:
So is this intended to do anything? I don't see a functional change. And shouldn't `defaultFont` be a `let`?